### ISSUE TITLE
MARCRelator properties removed in 3.0

### ISSUE THREAD

**Issue by dunn** [ruby-rdf/rdf-vocab#53]
The vocabularies were regenerated in https://github.com/ruby-rdf/rdf-vocab/commit/5a49cb06046495821edd57cf5af3660ebeda9e5e and `MARCRelators` lost all of its direct properties, so that `RDF::Vocab::MARCRelators.act` etc. now fail with `NoMethodError`:
```
undefined method `act' for RDF::StrictVocabulary(http://id.loc.gov/vocabulary/relators/):Class (NoMethodError)
```
@no-reply suggested this was likely unintentional and asked me to open an issue.

**Comment by gkellogg** [comment]
Sorry, the endpoint has not generated the vocabulary for a while, need a better location. I'll either find it and re-generate or revert.

**Comment by gkellogg** [comment]
Fixed in 481ad0146b83f7ff3bbc9fa56cab19b085c9fce7. Released in 3.0.2.

**Comment by dunn** [comment]
Thanks so much!
